app/controllers/login_controller.rb

```diff
@@ -7,11 +7,15 @@ def logout
   def public
     request.env['warden'].logout if github_authenticated? :private
     github_authenticate! :default
+    @user = gh.user
+    @emails = gh.user.emails
```

Comment:
Is there any validation that needs to happen here to make sure we're getting valid emails?